### PR TITLE
Support metrics for single model mode

### PR DESCRIPTION
**Pull request by mzegla**

### CODE REVIEW

File: bundle/manifests/intel.com_model_servers.yaml [hunk 1]
```diff
@@ -278,6 +278,17 @@ spec:
                     http_proxy:
                       description: http proxy to connect to the cloud storage
                       type: string
+                monitoring:
+                  type: object
+                  description: Configuration of service and model monitoring. This works only for single model mode. For serving multiple models, metrcs are enabled via configuration file.
```
Comment (rasapala) [inline bundle/manifests/intel.com_model_servers.yaml:283]: metrics are enabled

File: bundle_k8s/manifests/intel.com_model_servers.yaml [hunk 2]
```diff
@@ -278,6 +278,17 @@ spec:
                     http_proxy:
                       description: http proxy to connect to the cloud storage
                       type: string
+                monitoring:
+                  type: object
+                  description: Configuration of service and model monitoring. This works only for single model mode. For serving multiple models, metrcs are enabled via configuration file.
```
Comment (rasapala) [inline bundle_k8s/manifests/intel.com_model_servers.yaml:283]: metrics are enabled

File: config/crd/bases/intel.com_model_servers.yaml [hunk 3]
```diff
@@ -278,6 +278,17 @@ spec:
                     http_proxy:
                       description: http proxy to connect to the cloud storage
                       type: string
+                monitoring:
+                  type: object
+                  description: Configuration of service and model monitoring. This works only for single model mode. For serving multiple models, metrcs are enabled via configuration file.
```
Comment (rasapala) [inline config/crd/bases/intel.com_model_servers.yaml:283]: metrics are enabled

**Review by rasapala — CHANGES_REQUESTED** [review]

**Review by rasapala — APPROVED** [review]
LGTM

File: bundle/manifests/openvino-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -16,7 +16,7 @@ metadata:
             "name": "model-server-sample"
           },
           "spec": {
-            "image_name": "registry.connect.redhat.com/intel/openvino-model-server:2022.1",
+            "image_name": "registry.connect.redhat.com/intel/openvino-model-server:2022.3",
```
Comment (mzegla) [inline bundle/manifests/openvino-operator.clusterserviceversion.yaml:19]: replace with sha

File: bundle/manifests/intel.com_model_servers.yaml [hunk 5]
```diff
@@ -278,6 +278,17 @@ spec:
                     http_proxy:
                       description: http proxy to connect to the cloud storage
                       type: string
+                monitoring:
+                  type: object
+                  description: Configuration of service and model monitoring. This works only for single model mode. For serving multiple models, metrics are enabled via configuration file.
```
Comment (mzegla) [inline bundle/manifests/intel.com_model_servers.yaml:283]: add link to metrics docs

**Pull request merged**